vector.yaml

```diff
@@ -35,7 +36,10 @@ pipeline:
   - runs: |
       sed -i '7s/deny/allow/' src/lib.rs
       export RUSTFLAGS="$RUSTFLAGS --cap-lints=warn"
-      make build
+      # Force building with system provided openssl instead of vendored version
+      sed -i -E 's|^(openssl)\s*=.*version\s*=\s*"([0-9]+\.[0-9]+\.[0-9]+).*|openssl = { version = "0.10.72", default-features = false }|' Cargo.toml
```

Comment:
This is pinning to 0.10.72, whereas the capture group should be used here to always write back the regex detected version